Modules/FindProtobuf.cmake

```diff
@@ -197,10 +198,11 @@ function(PROTOBUF_GENERATE_PYTHON SRCS)
   foreach(FIL ${ARGN})
     get_filename_component(ABS_FIL ${FIL} ABSOLUTE)
```

Comment:
In the case that there is not a subdirectory path, this may generate double slashes.  Please revise to avoid that.